packages/formatters/src/global.ts

```diff
@@ -1,4 +1,4 @@
-declare const global: {};
+declare const global: unknown;
```

Comment:
have you tried `Record<unknown, unknown>` so typescript can still determine it as an `object` and reduce the side effects that it would do to us

Reply:
I can put `Record<string, unknown>` or just `object`. What do you think?

Reply:
I'd go with the Record. I think that's what typescript wants